.ci/script.bat

```diff
@@ -1,7 +1,7 @@
 call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -host_arch=amd64 -arch=amd64
 call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -test
 cd dist/msvc-compatible
-cl *.c /I ../kremlin/include /I . /I ../kremlin/kremlib/dist/minimal /c || goto :error
+cl *.c /I ../kremlin/include /I . /I ../kremlin/kremlib/dist/minimal /c /DEVERCRYPT_CAN_COMPILE_INTRINSICS /DEVERCRYPT_CAN_COMPILE_VALE /DEVERCRYPT_CAN_COMPILE_VEC128 /DEVERCRYPT_CAN_COMPILE_VEC256 || goto :error
```

Comment:
We should probably start using configure for Windows as well at some point. But that's for a different PR.

Reply:
yeah I don't know how to do that but happy to start talking about it!